addons/point_of_sale/static/src/app/screens/product_screen/product_screen.js

```diff
@@ -388,6 +388,15 @@ export class ProductScreen extends Component {
         return Array.from(new Set(products));
     }
 
+    getProductCartQty(product) {
+        const productTmplId = product.id;
+
+        return this.currentOrder.lines.reduce(
+            (acc, ol) => (ol.product_id.raw.product_tmpl_id === productTmplId ? acc + ol.qty : acc),
```

Comment:
Could you try not using raw but the path instead (it is more clear)
```suggestion
            (acc, ol) => (ol.product_id.product_tmpl_id.id === productTmplId ? acc + ol.qty : acc),
```

Reply:
@adgu-odoo Done!